packages/core/bootstrap/src/lib/middleware/burst-limit/index.ts

```diff
@@ -22,14 +22,21 @@ const availableSecondLimitCapacity = async (
   store: Store<BurstLimitState>,
   burstCapacity1s: number,
 ) => {
+  const enabled = parseBool(getEnv('RATE_LIMIT_CAPACITY'))
+  const defaultSecondslimit = 429
```

Comment:
Where is this number coming from?

Reply:
I think the ticket was referring to error code 429 and not 429 seconds. @boxhock can you confirm?

Reply:
Yes, referring to HTTP 429 error codes, not 429 seconds.